### PR TITLE
[stable/weave-scope] Add `read-only` mode (#6977)

### LINKED ISSUE #6977: Add `read-only` mode for weave scope
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): FEATURE REQUEST

**Which chart**: weave-scope 
Weave Scope has an option to disable potentially dangerous controls, for example 'delete pod' or 'run shell on node'. It would be good to leverage this feature in chart. 
Also, as per https://github.com/weaveworks/scope/issues/3153,  we can reduce the number of perms given to cluster role (and thus, optionally, disable delete and update permissions).

### PR DESCRIPTION
**Pull request by ivanfitenko**
* add option to disable controls (e.g. start/stop containers, terminals, logs etc)
* use more granular permissions for cluster role
* use newer weave scope release
* bump chart version
* remove compatibility note for EOL kubernetes versions and older weave scope versions

**What this PR does / why we need it**: implements features requested in #6977

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #6977

 **Special notes for your reviewer:** @omkensey please review

### CODE REVIEW

**Review by unguiculus — CHANGES_REQUESTED** [review]
Please bump the minor version.

**Comment by unguiculus** [comment]
cc @omkensey

File: stable/weave-scope/charts/weave-scope-agent/templates/clusterrole.yaml [hunk 1]
```diff
@@ -11,14 +11,61 @@ metadata:
...
+      - deployments
+      - daemonsets
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - extensions
+    resources:
+      - deployments/scale
```
Comment (omkensey) [inline stable/weave-scope/charts/weave-scope-agent/templates/clusterrole.yaml:66]: Are these permissions the ones officially supported by WeaveWorks?  Originally I took the permissions for the chart from the output of https://cloud.weave.works/k8s/scope.yaml?k8s-version=[version] (where [version] was 1.6.something].  If so, the annotations in the _helpers.tpl should be updated as well.

File: stable/weave-scope/charts/weave-scope-agent/templates/daemonset.yaml [hunk 2]
```diff
@@ -34,6 +34,9 @@ spec:
             - '--probe.docker.bridge={{ .Values.dockerBridge }}'
             - '--probe.docker=true'
             - '--probe.kubernetes=true'
+            {{ if .Values.disableFrontendControls }}
+            - "-probe.no-controls"
```
Comment (omkensey) [inline stable/weave-scope/charts/weave-scope-agent/templates/daemonset.yaml]: Double-dash here to match the style of the other arguments, though I wouldn't do a commit just for that if this works.

File: stable/weave-scope/charts/weave-scope-agent/templates/clusterrole.yaml (hunk 1 above)
Comment (ivanfitenko) [reply stable/weave-scope/charts/weave-scope-agent/templates/clusterrole.yaml:66]: Yes, these were official ones for version 1.9.dont_remember_now. Now that the latest version is 1.10, it's a good time to update the permissions once again.

**Comment by ivanfitenko** [comment]
It appears that this pull request was misplaced somewhat. I will create another one in a proper form and proceed working on the feature. Closing this one

**Comment by davidkarlsen** [comment]
@ivanfitenko Did you place a new one?

**Comment by davidkarlsen** [comment]
@ivanfitenko ping?

**Pull request closed without merging**